### PR TITLE
Navbar for Teams/Decks/Settings info

### PR DESCRIPTION
**Pull request by hank0cean**
A little css

### CODE REVIEW

File: src/components/card.jsx [hunk 1]
```diff
@@ -4,6 +4,12 @@ import { Button } from '@material-ui/core'
 import './../styles/card.css'
 
 class Card extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+
+    }
```
Comment (blockco) [inline src/components/card.jsx:11]: useless constructor

File: src/components/decksDropdown.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function DecksDropdown(props) {
+
+  return ( 
+    ((props.itemOpen &&
+    <div className="decksDropdown">
+      <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Sed unde eius consequuntur sequi praesentium in tempore alias! Dolores doloribus accusantium incidunt, maiores molestias explicabo quos assumenda debitis perspiciatis quas voluptates?</p>
+    </div>) || null)
+  );
```
Comment (blockco) [inline src/components/decksDropdown.jsx:10]: LGTM but weird

File: src/components/teamsDropdown.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+function TeamsDropdown(props) {
+  
+  return ( 
+    ((props.itemOpen &&
+    <div className="teamsDropdown">
+      <div className="teamsInfo">
+        <div className="redTeam">
+          Red Team
+          <p>--------</p>
```
Comment (blockco) [inline src/components/teamsDropdown.jsx:12]: use a component for this

File: src/components/card.jsx (hunk 1 above)
Comment (hank0cean) [reply src/components/card.jsx]: delete

File: src/components/teamsDropdown.jsx (hunk 3 above)
Comment (hank0cean) [reply src/components/teamsDropdown.jsx]: done

**Pull request merged**